overlord/devicestate/firstboot20_test.go

```diff
@@ -118,12 +117,14 @@ volumes:
 }
 
 func (s *firstBoot20Suite) TestPopulateFromSeedCore20Happy(c *C) {
-	r := boottest.ForceModeenv("", &boot.Modeenv{
+	m := boot.Modeenv{
 		Mode:           "run",
 		RecoverySystem: "20191018",
 		Base:           "core20_1.snap",
-	})
-	defer r()
+	}
+	err := m.Write("")
+	c.Assert(err, IsNil)
+	defer os.Remove(dirs.SnapModeenvFileUnder(dirs.GlobalRootDir))
```

Comment:
(nitpick) I don't think we need this, the firstBootBaseTest runs dirs.SetRootDir(tmpdir) and that tmpdir is created via c.MkDir() so it's automatically cleaned up.